code/ndarray.h

```diff
@@ -28,6 +28,7 @@
 
 #if CIRCUITPY
 #define mp_obj_is_bool(o) (MP_OBJ_IS_TYPE((o), &mp_type_bool))
+#define mp_obj_is_int(x) (MP_OBJ_IS_INT((x)))
```

Comment:
I agree, this is the easiest way. MicroPython was just trying to canonicalize their macro names.

Reply:
I didn't mean for this to be a review, just a comment.

Reply:
Oh, but a comment by the oracle qualifies as a review. Seriously, Dan, I really appreciate that you chipped in.

Reply:
I am only an oracle because I subscribe to all the micropython git notifcations :smile: